core/web/log_controller.go

```diff
@@ -0,0 +1,101 @@
+package web
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"go.uber.org/zap/zapcore"
+)
+
+// LogController manages the logger config
+type LogController struct {
+	App chainlink.Application
+}
+
+type LoglevelPatchRequest struct {
+	LogLevel string `json:"logLevel"`
```

Comment:
Omit the prefix as it is encapsulated by the LogRequest
```suggestion
	Level string `json:"level"`
```